cfn-resources/cloud-backup-snapshot/cmd/resource/resource.go

```diff
@@ -18,24 +18,24 @@ import (
 	"context"
 	"fmt"
 	"strconv"
-
-	progressEvents "github.com/mongodb/mongodbatlas-cloudformation-resources/util/progressevent"
+	"strings"
 
 	"github.com/aws-cloudformation/cloudformation-cli-go-plugin/cfn/handler"
+	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/service/cloudformation"
 	"github.com/mongodb/mongodbatlas-cloudformation-resources/util"
 	"github.com/mongodb/mongodbatlas-cloudformation-resources/util/constants"
 	"github.com/mongodb/mongodbatlas-cloudformation-resources/util/logger"
+	progressEvents "github.com/mongodb/mongodbatlas-cloudformation-resources/util/progressevent"
 	"github.com/mongodb/mongodbatlas-cloudformation-resources/util/validator"
-	"github.com/openlyinc/pointy"
 	"github.com/spf13/cast"
 	"go.mongodb.org/atlas/mongodbatlas"
 )
 
-var CreateRequiredFields = []string{constants.PubKey, constants.PvtKey, constants.ClusterName, constants.GroupID}
-var DeleteRequiredFields = []string{constants.PubKey, constants.PvtKey, constants.ClusterName, constants.GroupID, constants.SnapshotID}
-var ReadRequiredFields = []string{constants.PubKey, constants.PvtKey, constants.GroupID, constants.SnapshotID}
-var ListRequiredFields = []string{constants.PubKey, constants.PvtKey, constants.GroupID}
+var CreateRequiredFields = []string{constants.ClusterName, constants.GroupID}
+var DeleteRequiredFields = []string{constants.ClusterName, constants.GroupID, constants.SnapshotID}
+var ReadRequiredFields = []string{constants.GroupID, constants.SnapshotID}
+var ListRequiredFields = []string{constants.GroupID}
```

Comment:
Can we standardise on `ProjectID` which is how we usually refer to these in our docs

```suggestion
var ReadRequiredFields = []string{constants.ProjectID, constants.SnapshotID}
var ListRequiredFields = []string{constants.ProjectID}
```

Reply:
Changed to ProjectId